### ISSUE TITLE
Create new user account for frontend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#15]
## User story
1. As a {HR-employee}
2. I want / need {to be able to create a new user account (with employee role) for the frontend}
3. So that {new HR-employees can easily be onboarded and use the software as well}

## Acceptance criteria
* new account can be created by inserting an email address where an automatically generated password is sent to
* password needs to be changed at first login of new user
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
